mrp_product_use_case/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+====================
+MRP Product Use Case
```

Comment:
IMHO the name is not very clear, something like `mrp_bom_product_find` would be more descriptive of the functionality, I find "Use Case" very misleading.

Reply:
Yes name can be improved as suggested or

mrp_component_use_case or mrp_bom_component_use_case or mrp_component_find

Reply:
I like mrp_component_find, because this is exactly what the module is: a wizard to find components on the manufacturing app (mrp).

Reply:
I see that there is a module called `mrp_bom_component_menu` already in OCA so I will follow that structure also.  What do you thing of `mrp_bom_component_find` for this module?

Reply:
+1 for mrp_bom_component_find

Reply:
@pedrobaeza what do you think of this name ?